test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +77,19 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter' {
+            $job = Start-Job -ScriptBlock { $pwd } -WorkingDirectory $TestDrive | Wait-Job
+            $jobOutput = Receive-Job $job
+            $jobOutput | Should -BeExactly $TestDrive.ToString()
+        }
+
+        It 'Throws an error when the working directory parameter is <case>' -TestCases $invalidPathTestCases {
+            foreach ($tempPath in $invalidPaths)
```

Comment:
This should add 
```powershell
param($path, $case)
```

Then you do not need the `foreach`, just use `$path`

Reply:
fixed in 5a5efd4f1d00333cea56bdc09f9c4891654afa7f